process/smartContract/builtInFunctions/keyValueStorage_test.go

```diff
@@ -0,0 +1,54 @@
+package builtInFunctions
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/data/state"
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/ElrondNetwork/elrond-go/process/mock"
+	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSaveKeyValue_ProcessBuiltinFunction(t *testing.T) {
+	t.Parallel()
+
+	coa := saveKeyValueStorage{
+		funcGasCost: 1,
+		gasConfig: BaseOperationCost{
+			StorePerByte:    1,
+			ReleasePerByte:  1,
+			DataCopyPerByte: 1,
+			PersistPerByte:  1,
+			CompilePerByte:  1,
+		},
+	}
+
+	addr := []byte("addr")
+
+	acc, _ := state.NewUserAccount(mock.NewAddressMock(addr))
```

Comment:
Add a test to see that the value was stored in the account.

Reply:
added.